Makefile

```diff
@@ -12,96 +18,96 @@ build-and-push-gitea: build-gitea-image push-gitea-image
 build-and-push-guacamole: build-guacamole-image push-guacamole-image
 
 bootstrap:
-	echo -e "\n\e[34m»»» 🧩 \e[96mBootstrap Terraform\e[0m..." \
+	$(call target_title, "Bootstrap Terraform") \
```

Comment:
whats this change

Reply:
I started to use recipes (macros) to write code once and reuse as needed. I'm working on other cases but it's not ready yet.

Reply:
Line 10

> target_title = @echo -e "\n\e[34m»»» 🧩 \e[96m$(1)\e[0m..."